src/extensions/registries/page-registry.ts

```diff
@@ -1,93 +1,95 @@
 // Extensions-api -> Custom page registration
-import type { PageMenuTarget } from "./page-menu-registry";
 import type React from "react";
+import type { UrlParam } from "../../renderer/navigation/url-param";
+
 import path from "path";
 import { action } from "mobx";
-import { compile } from "path-to-regexp";
 import { BaseRegistry } from "./base-registry";
 import { LensExtension, sanitizeExtensionName } from "../lens-extension";
 import logger from "../../main/logger";
-import { rectify } from "../../common/utils";
 
 export interface PageRegistration {
   /**
-   * Page ID or additional route path to indicate uniqueness within current extension registered pages
-   * Might contain special url placeholders, e.g. "/users/:userId?" (? - marks as optional param)
+   * Page-id, part of of extension's page url, must be unique within same extension
```

Comment:
The dash makes it look like it is one concept, which is slightly confusing when documenting something called "id".

```suggestion
   * Page's `id`, part of of extension's page url, must be unique within same extension
```